script.js

```diff
@@ -0,0 +1,170 @@
+// Remove and complete icons in SVG format
+var removeSVG = ' <span class="material-symbols-outlined">delete</span>';
+var completeSVG = '<span class="material-symbols-outlined">task_alt</span>';
+
+const taskTitel = document.getElementById("task-titel");
+const taskAssign = document.getElementById("task-assign");
+const taskSearch = document.getElementById("task-search");
+const addBtn = document.getElementById("add");
+const clearBtn = document.getElementById("clear-btn");
+const todos = document.getElementById("todos");
+const todo = document.getElementById("todo");
+const done = document.getElementById("done");
+clearBtn.addEventListener("click", clearSearch);
+add.addEventListener("click", addTask);
+taskSearch.addEventListener("keyup", filterTasks);
+
+let tasks = [];
+
+function getTaskFromLocalStorage() {
+  let reTasks = JSON.parse(localStorage.getItem("tasks"));
+  if (reTasks == null) {
+    tasks = [];
+  } else {
+    tasks = reTasks;
+  }
+}
+
+function updateLocalStorage() {
+  let taskString = JSON.stringify(tasks);
```

Comment:
`const tasks = JSON.stringify(tasks);`

Reply:
can't assign the same value (tasks)
ok 
am making some changes